### PR TITLE
fix(autosuggest): incorrect mark style

### PR DESCRIPTION
**Pull request by bualoy-napat**
## Description

`mark` color on suggestion list is not correct when attach the `ef-autosuggest` inside the shadow DOM. The root cause of this issue is that the `mark` element is not within the scope of the `ef-autosuggest` theme. It is slotted inside the `ef-item`. Therefore, moving the `mark` style to the scope of `ef-item` will fix the issue.

Fixes # (issue)
ELF-2366

## Checklist

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] I have checked my code and corrected any misspellings

### CODE REVIEW

**Review by kosol-komenek-rft — DISMISSED** [review]
Look good for me.

File: packages/elements/src/autosuggest/__demo__/index.html [hunk 1]
```diff
@@ -665,66 +667,21 @@
...
-                },
-                {
-                  suggest: 'Cornelius Martin'
-                }
-              ]
-            }
-          };
-        })();
+        const wrapper = document.getElementById('search-suggestion');
+        let searchField = document.createElement('ef-search-field');
```
Comment (wattachai-lseg) [inline packages/elements/src/autosuggest/__demo__/index.html]: let's replace `let` with `const` if there is no reassignment.

**Review by wattachai-lseg — CHANGES_REQUESTED** [review]
2 more things:
* Could you replace relative path import (`import <something> from './../../../lib/autosuggest/<path>`) with module import (`import <something> from @refinitiv-ui/elements/autosuggest/<path>`) instead?
* I couldn't find a description for "inside shadow root" demo we discussed earlier.

File: packages/elements/src/autosuggest/__demo__/index.html (hunk 1 above)
Comment (bualoy-napat) [reply packages/elements/src/autosuggest/__demo__/index.html]: Updated!

File: packages/elements/src/autosuggest/__demo__/index.html [hunk 2]
```diff
@@ -665,66 +667,25 @@
...
+        const searchField = document.createElement('ef-search-field');
+        const autoSuggest = document.createElement('ef-autosuggest');
+        autoSuggest.attach = searchField;
+        autoSuggest.addEventListener('suggestions-fetch-requested', (event) => {
+          const query = event.detail.query;
+          window.filterData(query).then(({ data }) => {
+            autoSuggest.suggestions = data;
+          });
+        });
+        wrapper.appendChild(searchField);
```
Comment (wattachai-lseg) [inline packages/elements/src/autosuggest/__demo__/index.html]: Elements should be appended to wrapper element's shadow root not the element itself!

**Review by wattachai-lseg — APPROVED** [review]

**Pull request merged**